demisto_sdk/commands/common/content_constant_paths.py

```diff
@@ -11,3 +11,16 @@
 DEFAULT_ID_SET_PATH = CONTENT_PATH / "Tests" / "id_set.json"
 MP_V2_ID_SET_PATH = CONTENT_PATH / "Tests" / "id_set_mp_v2.json"
 XPANSE_ID_SET_PATH = CONTENT_PATH / "Tests" / "id_set_xpanse.json"
+
+
+PYTHONPATH = [
+    Path(CONTENT_PATH),
+    Path(CONTENT_PATH / "Packs" / "Base" / "Scripts" / "CommonServerPython"),
+    Path(CONTENT_PATH / "Tests" / "demistomock"),
+]
+try:
+    PYTHONPATH.extend(
+        dir for dir in Path(CONTENT_PATH / "Packs" / "ApiModules" / "Scripts").iterdir()
+    )
+except FileNotFoundError:
+    pass
```

Comment:
when does this happen? Should we log it?

Reply:
It happens when The ApiModules directory is missing from the CONTENT_PATH. In this case, it won't help to add this to PYTHONPATH.

Reply:
let's log it then, at least as `debug`